collect_stata/dataset.py

```diff
@@ -9,41 +17,37 @@ class Dataset:
     """
     Dataset allows the user to read, test and export data in different formats.
 
-    Example:
-
-        dataset = Dataset()
-        dataset.read_stata("../input/dataset.dta")
-        dataset.write_json("../output/dataset.json")
+    Args:
+        dataset: Contains the data in tabular format.
+        metadata: Contains the metadata in json format.
     """
 
     def __init__(self):
         self.dataset = None
         self.metadata = None
 
-    def read_stata(self, dta_name):
-        """
-        Function to read data in stata format.
-
-        Parameter:
+    def read_stata(self, dta_name: str):
+        """Method to read data in stata format.
 
-        dta_name: Name of the data in stata format
+        Args:
+            dta_name (str): Name of the data in stata format.
```

Comment:
Please use type hints instead. For all functions. Then remove the type from the docstrings.